### ISSUE TITLE
WikiOnBoard (Symbian3) does not work with the new dumps.

### ISSUE THREAD

**Issue by adam-910** [cip/WikiOnBoard#87]
Dear Cip,
thank You very much for your program and for supporting Symbian! 

Unfortunately, the new Wikipedia-dumps program does not see. I tried to install wikipedia_ru_15G_2015-03_all and wikipedia_nl_13G_2015-05_all, but to no avail. I use an old dump (2014). I have a Nokia 808 Symbian^3 FP2. 
Is it possible to help solve the problem?

Thank you.

**Comment by jupelton** [comment]
I can second this problem: Wikionboard 3 on symbian won't open newest zim-files

**Comment by adam-910** [comment]
"I can second this problem: Wikionboard 3 on symbian won't open newest zim-files" This is bad. Maybe there is a way to convert new zim-files to the old? If you know this, please indicate the relevant link.

**Comment by huellif** [comment]
Do you have an idea what got changed or what's the problem?

**Comment by adam-910** [comment]
No, any ideas about this I don't have. The problem is not solved.

**Comment by Ladanifer** [comment]
The same problem here. 
What a pity! It's a very useful app.

**Comment by BerriUQAM** [comment]
I know the app is pretty much abandoned now but anyways... It is also not working with WikiOnBoard 0.0.37 on my N95 8GB. It properly opens zim files and can list entries but throws an error upon opening one (see screenshots below).

Does not work with wiktionary_fr_all_nopic_2016-04.zim, but does work okay with wiktionary_fr_all_12_2013.

![scr000004](https://cloud.githubusercontent.com/assets/4924563/19948600/a93abee0-a14d-11e6-97b0-ab94a9f4ce54.png) ![scr000005](https://cloud.githubusercontent.com/assets/4924563/19948599/a9390c08-a14d-11e6-9e25-4a50c1d76523.png)
